src/store/modules/api.js

```diff
@@ -80,11 +80,9 @@ export default {
     },
 
     addRiskFactors({commit}, {riskFactors, data}) {
-      data.find((factor) => {
-        if (riskFactors.includes(factor.id)) {
-          commit('PUSH_RISK_FACTOR', factor);
-        }
-      });
+      data
+        .filter((factor) => riskFactors.includes(factor.id))
+        .map((factor) => commit('PUSH_RISK_FACTOR', factor));
```

Comment:
Ciekawe podejście, ale ani stara ani nowa wersja mi się w pełni nie podoba ;)
Odnośnie nowej:
1. filter jest dobym pomysłem ale tworzy zbędnie nową tablicę
2. map troszeczkę do czego innego służy, możemy go wykorzystać w ten sposób ale kłóci się to z jego ideą.

Myśle że najoptymalniejsza wersja to jest ta pierwotna ale find powinien być zamieniony na forEach. Bo ten find też jest troszkę nadmiarowo użyty skoro chcemy tylko przeiterować po liście.

Reply:
Prawda, forEach byłby dużo lepszy, niepotrzebnie iteruje dwa razy.
Dziwi mnie natomiast, że po zmianie oraz braku `return` nie triggeruje się error eslinta.

PS. dlaczego nie `for .. of`?